### ISSUE TITLE
Training Set 의 예측결과가 부정확한 문제

### ISSUE THREAD

**Issue by penny4860** [penny4860/SVHN-Single-Shot-Detector#10]
* 100개의 train set 으로 test 해본 결과 학습은 어느정도 되고 있다.
* 하지만, Training set 의 bounding box 의 예측이 부정확한 문제가 있다.
  * Train set 을 늘려서 Test
      * 100개가 너무 적나?
  * Network 의 Complexity 를 높인다.
  * Batch Norm Test
  * Transfer Learning Test
